redaxo/src/core/lib/clang/clang.php

```diff
@@ -183,12 +183,14 @@ public function getValue($value)
     /**
      * Counts the clangs.
      *
+     * @param bool $ignoreOfflines
+     *
      * @return int
      */
-    public static function count()
+    public static function count($ignoreOfflines = false)
     {
         self::checkCache();
```

Comment:
hier könnte der cache-check jetzt weg?

Reply:
Ich denke schon, ist aber auch in der Methode `getAllIds` vorhanden. Deshalb hatte ihn jetzt mit drin gelassen.